test/fixtures/rules/std_rule_cases/LT02-indent.yml

```diff
@@ -1941,3 +1941,32 @@ test_fail_issue_4680:
         {% else %}
         col1 > 0
       {% endif %}
+
```

Comment:
can you add a test with a fail_str?

Reply:
@WittierDinosaur I replaced the `pass_str` with `fix_str` and added a `fail_str` with the unwanted indentation